Description: bsd rand didn't work, use normal rand
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/intro.cpp
+++ b/src/intro.cpp
@@ -135,10 +135,12 @@
 //######################################
 
 /** Declare random number function. */
-#define bsd_rand bsd_rand_actual
+//#define bsd_rand bsd_rand_actual
+#define bsd_rand rand
 
 /** Declare random number function. */
-#define bsd_srand bsd_srand_actual
+//#define bsd_srand bsd_srand_actual
+#define bsd_srand srand
 
 /** Declare which color space to use. */
 #define COLOR_SPACE_THEFLASH 0
